rero_ils/config.py

```diff
@@ -116,42 +116,54 @@ def _(x):
     ('de', _('German')),
     ('it', _('Italian')),
 ]
+# Define the default system currency in used. Each organisation can override
+# this parameter using the 'default_currency' field
+RERO_ILS_DEFAULT_CURRENCY = 'CHF'
```

Comment:
Is it really related to this commit?

Reply:
It is. I added all missing configuration from Invenio to RERO-ils in order to avoid potential problems about the migration.